debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -89,6 +91,19 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
         final Charset databaseCharset;
         try (PostgresConnection tempConnection = new PostgresConnection(connectorConfig.getJdbcConfig(), PostgresConnection.CONNECTION_GENERAL)) {
             databaseCharset = tempConnection.getDatabaseCharset();
+
```

Comment:
@mfvitale Should not this check go to `io.debezium.connector.postgresql.PostgresConnector.validateConnection(Map<String, ConfigValue>, Configuration)`?

Reply:
Definitely a better place. Thanks.